docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst

```diff
@@ -138,7 +138,7 @@ Services
 +                    +---------------------------------------------------------------------------------------------------+-------------+-------------------------------------------------------------------------------------+
 |                    | :mod:`OptimizationMind <DIRAC.WorkloadManagementSystem.Service.OptimizationMindHandler>`          |             |                                                                                     |
 +                    +---------------------------------------------------------------------------------------------------+-------------+-------------------------------------------------------------------------------------+
-|                    | :mod:`PilotManager <DIRAC.WorkloadManagementSystem.Service.PilotManagerHandler>`                  | YES         |                                                                                     |
+|                    | :mod:`PilotManager <DIRAC.WorkloadManagementSystem.Service.PilotManagerHandler>`                  | YES         | In case there are HTCondor CEs to deal with, the HTCondor "working directory" should exist and be accessible on each and every machine. The path is available in the configuration (/Resources/Computing/HTCondorCE/WorkingDirectory). The default one is hardcoded in the source code of the HTCondorCE itself.                                                                                      |
```

Comment:
This overlong line doesn't work. The table boundaries need to be all aligned and the same for all rows.
You have to break this into multiple lines

You could link to https://dirac.readthedocs.io/en/latest/AdministratorGuide/Configuration/ConfReference/Resources/Computing/index.html#htcondorce-parameters

Though maybe it would be better to replace YES with PARTIAL and have the explanation in the HTCondorCE documentation.